### DISCUSSION TITLE
[Idea] Add gitignore support for Jetbrains IDE

### DISCUSSION THREAD

**Discussion by Dreimus** [PrestaShop/PrestaShop#35382] [Ideas]
### The problem

When I contribute, I always have .idea folder content considered as versioned files. Which makes harder branch switches.

### The solution

Ignore .idea file in root .gitignore file

### Do you plan to work on this subject?

- [X] I'm willing to contribute a [formal specification](https://github.com/PrestaShop/prestashop-specs).
- [ ] I'm willing to provide any wireframes or design assets required for this feature.
- [X] I'm willing to submit a Pull Request that implements this feature.
- [ ] I'm willing to help verify that the implemented feature works as intended and produces no unintended side effects.

**Comment by M0rgan01** [comment]
Hello, this configuration is linked to your environment, I don't think that added to the project is the right place. You can exclude the folder globally, by right clicking on .idea directory -> git -> add to .gitignore -> .git/info/exclude
